acceptance/testdata/detector/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:bionic
+FROM ubuntu:jammy
```

Comment:
This change causes https://github.com/buildpacks/lifecycle/blob/1a1de0850a2e8cd34bedccf9856886077dd702e4/acceptance/detector_test.go#L163 to start failing when 

```
[[stacks]]
id = "io.buildpacks.stacks.bionic"

[[stacks]]
id = "io.buildpacks.stacks.jammy"
```
 is added to acceptance/testdata/detector/container/cnb/buildpacks/simple_buildpack/simple_buildpack_version/buildpack.toml without corresponding changes in how we read the descriptor